help/destinations/catalog/email-marketing/mailchimp-tags.md

```diff
@@ -0,0 +1,199 @@
+
+### Prerequisites for the [!DNL Mailchimp Tags] destination {#prerequisites-destination}
+
+Note the following prerequisites in order to export data from Platform to your [!DNL Mailchimp Tags] account:
+
+#### You need to have a [!DNL Mailchimp] account {#prerequisites-account}
+
+Before you can create a [!DNL Mailchimp Tags] destination, you must first ensure that you have a [!DNL Mailchimp] account. If you do not have one already visit the [[!DNL Mailchimp] signup page](https://login.mailchimp.com/signup/) to register and create your account.
+
+#### Gather [!DNL Mailchimp] API key {#gather-credentials}
+
+You need your [!DNL Mailchimp] **API key** to authenticate the [!DNL Mailchimp Interest Categories] destination against your [!DNL Mailchimp] account. The **API key** serves as the **Password** when you [authenticate the destination](#authenticate).
+
+If you do not have your **API key**, Sign in to your account and refer to the [[!DNL Mailchimp] Generate your API key](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key) documentation to create one. 
+
+An example of an API key is `0123456789abcdef0123456789abcde-us14`. 
+
+>[!IMPORTANT]
+>
+>If you generate the **API key**, write it down as you will not be able to access it after generation.
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
+
+Next, you must identify your [!DNL Mailchimp] data center. To do this, log in to your [!DNL Mailchimp] account and navigate to the **API keys section** of your account. 
+
+The value is the first part of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center is `us14`. 
+
+It's also appended to your API key in the form *key-dc*; if your API key is `0123456789abcdef0123456789abcde-us14`, then the data center is `us14`. 
+
+Write down the data center value *(`us14` in this example)*, you need this value when you [fill in destination details](#destination-details).
+
+If you require further guidance, refer to the [[!DNL Mailchimp] Fundamentals documentation](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-structure).
+
+### Guardrails {#guardrails}
+
+Refer to the [!DNL Mailchimp] [rate limits](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-limits) for detailed information about the limits imposed by the [!DNL Mailchimp] API.
+
+## Supported identities {#supported-identities}
+
+[!DNL Mailchimp] supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+| Email | Contact email Address | Mandatory |
+
+{style="table-layout:auto"}
+
+## Supported audiences {#supported-audiences}
+
+This section describes which type of audiences you can export to this destination.
+
+| Audience origin | Supported | Description | 
```

Comment:
Please update the supported audiences table to:


| Audience origin | Supported | Description | 
---------|----------|----------|
| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md).|
| Custom uploads | X | Audiences [imported](../../../segmentation/ui/overview.md#import-audience) into Experience Platform from CSV files. |